scripts/mem_usage.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
I didn't  found nothing suspect, but are you sure that this script will work both on python2 and on python3?

Reply:
Actually, it does not ;-) but I will fix it.